### PR TITLE
Update permissions

### PR DESCRIPTION
**Pull request by MinnDevelopment**
[contributing]: https://jda.wiki/contributing/contributing/

## Pull Request Etiquette

- [x] I have checked the PRs for upcoming features/bug fixes.
- [x] I have read the [contributing guidelines][contributing].

### Changes

- [ ] Internal code
- [x] Library interface (affecting end-user code) 
- [ ] Documentation
- [ ] Other: \_____ 

Closes Issue: NaN

## Description

This updates the permission enum and replaces (through deprecation) the old `Permission.MANAGE_EMOJIS_AND_STICKERS` permission with the new `Permission.MANAGE_GUILD_EXPRESSIONS` permission.

### References
- https://github.com/discord/discord-api-docs/pull/6017
- https://github.com/discord/discord-api-docs/pull/6039
- https://github.com/discord/discord-api-docs/pull/6082

### CODE REVIEW

File: src/main/java/net/dv8tion/jda/api/Permission.java [hunk 1]
```diff
@@ -68,15 +75,17 @@
...
+    VOICE_STREAM(              9, true, true, "Video"),
+    VOICE_CONNECT(            20, true, true, "Connect"),
+    VOICE_SPEAK(              21, true, true, "Speak"),
+    VOICE_MUTE_OTHERS(        22, true, true, "Mute Members"),
+    VOICE_DEAF_OTHERS(        23, true, true, "Deafen Members"),
+    VOICE_MOVE_OTHERS(        24, true, true, "Move Members"),
+    VOICE_USE_VAD(            25, true, true, "Use Voice Activity"),
+    VOICE_START_ACTIVITIES(   39, true, true, "Launch Activities in Voice Channels"),
+    VOICE_USE_SOUNDBOARD(     42, true, true, "Use Soundboard in Voice and Stage Channels"),
+    VOICE_USE_EXTERNAL_SOUNDS(45, true, true, "Use Soundboards from other Servers"),
```
Comment (MinnDevelopment) [inline src/main/java/net/dv8tion/jda/api/Permission.java:88]:
```suggestion
    VOICE_USE_EXTERNAL_SOUNDS(45, true, true, "Use External Sounds"),
```

File: src/main/java/net/dv8tion/jda/api/Permission.java [hunk 2]
```diff
@@ -68,15 +75,17 @@
...
+    PRIORITY_SPEAKER(          8, true, true, "Priority Speaker"),
+    VOICE_STREAM(              9, true, true, "Video"),
+    VOICE_CONNECT(            20, true, true, "Connect"),
+    VOICE_SPEAK(              21, true, true, "Speak"),
+    VOICE_MUTE_OTHERS(        22, true, true, "Mute Members"),
+    VOICE_DEAF_OTHERS(        23, true, true, "Deafen Members"),
+    VOICE_MOVE_OTHERS(        24, true, true, "Move Members"),
+    VOICE_USE_VAD(            25, true, true, "Use Voice Activity"),
+    VOICE_START_ACTIVITIES(   39, true, true, "Launch Activities in Voice Channels"),
+    VOICE_USE_SOUNDBOARD(     42, true, true, "Use Soundboard in Voice and Stage Channels"),
```
Comment (MinnDevelopment) [inline src/main/java/net/dv8tion/jda/api/Permission.java:87]:
```suggestion
    VOICE_USE_SOUNDBOARD(     42, true, true, "Use Soundboard"),
```

File: src/main/java/net/dv8tion/jda/api/Permission.java [hunk 3]
```diff
@@ -68,15 +75,17 @@
...
-    VOICE_START_ACTIVITIES(39, true, true, "Launch Activities in Voice Channels"),
+    PRIORITY_SPEAKER(          8, true, true, "Priority Speaker"),
+    VOICE_STREAM(              9, true, true, "Video"),
+    VOICE_CONNECT(            20, true, true, "Connect"),
+    VOICE_SPEAK(              21, true, true, "Speak"),
+    VOICE_MUTE_OTHERS(        22, true, true, "Mute Members"),
+    VOICE_DEAF_OTHERS(        23, true, true, "Deafen Members"),
+    VOICE_MOVE_OTHERS(        24, true, true, "Move Members"),
+    VOICE_USE_VAD(            25, true, true, "Use Voice Activity"),
+    VOICE_START_ACTIVITIES(   39, true, true, "Launch Activities in Voice Channels"),
```
Comment (MinnDevelopment) [inline src/main/java/net/dv8tion/jda/api/Permission.java:86]:
```suggestion
    VOICE_START_ACTIVITIES(   39, true, true, "Use Activities"),
```

File: src/main/java/net/dv8tion/jda/api/Permission.java [hunk 4]
```diff
@@ -60,6 +67,7 @@
     MESSAGE_EXT_EMOJI(       18, true, true, "Use External Emojis"),
     USE_APPLICATION_COMMANDS(31, true, true, "Use Application Commands"),
     MESSAGE_EXT_STICKER(     37, true, true, "Use External Stickers"),
+    VOICE_MESSAGE_SEND(      46, true, true, "Send Voice Messages"),
```
Comment (MinnDevelopment) [inline src/main/java/net/dv8tion/jda/api/Permission.java:70]: This is likely gonna be confusing. People might interpret this as text-in-voice instead.

Comment (Chew) [reply src/main/java/net/dv8tion/jda/api/Permission.java:70]: "Send Audio Message" then? Since bots don't have voices 😄

**Pull request merged**